cloudformation/infrastructure/load-balancers.yaml

```diff
@@ -32,27 +28,6 @@ Resources:
             Tags: 
                 - Key: Name
                   Value: !Ref EnvironmentName
-
-    LoadBalancerListener:
```

Comment:
This looks like it should be a nice simplification since that will avoid the possibility of anything ending up in a partially-deployed state with the default listener